pom.xml

```diff
@@ -26,7 +26,7 @@
   (in particular, if you are doing it for the first time), instead do
   'mvn package'.  If you are interested in the full story, see
   https://issues.apache.org/jira/browse/HBASE-6795.
-
+For triggering test
```

Comment:
Nope. I add a new commit for triggering the UTs in other modules. Otherwise it will only run the UTs in hbase-common module. You can see the first 3 pre commit result posts here...

When merging I will revert this commit first.